### ISSUE TITLE
Is the weighted error equation correct ?

### ISSUE THREAD

**Issue by pshroff04** [detectRecog/CCPD#19]
loss += 0.8 * nn.L1Loss().cuda()(fps_pred[:][:2], y[:][:2])
 loss += 0.2 * nn.L1Loss().cuda()(fps_pred[:][2:], y[:][2:])

With reference to above lines? I understand you are taking .8 times loss of first two examples ?? and .2 times loss of other examples in batch. Is it what you are trying to do ? Please clarify on this.
